OpenRA.Mods.Common/Activities/Air/Fly.cs

```diff
@@ -57,6 +57,9 @@ public static void FlyToward(Actor self, Aircraft plane, int desiredFacing, WDis
 
 		public override Activity Tick(Actor self)
 		{
+			// Refuse to take off if it would land immediately again.
+			if (plane.ForceLanding) return null;
```

Comment:
Returning null bypasses the rest of the activity queue without giving those activities any chance to clean up.  This can break things in arbitrarily bad ways.

Requesting guidance from @obrakmann.